test/functional/fixtures/api/es-next/eval/test.js

```diff
@@ -24,15 +24,15 @@ describe('[API] t.eval', function () {
     it('Should have the correct callsite if an error occurs on instantiation', function () {
         return runTests('./testcafe-fixtures/eval-test.js', 'Error on instantiation', { shouldFail: true })
             .catch(function (errs) {
-                expect(errs[0]).contains('Client code is expected to be specified as a function, but "string" was passed.');
+                expect(errs[0]).contains('Hybrid function code is expected to be specified as a function, but "string" was passed.');
```

Comment:
I like more previous text for error `'Client code is expected to be specified as a function, but "string" was passed.'`
Because now it look strange, we no longer have term 'Hybrid ' in API. I think we should not use it.
May be we can get rid of `Hybrid ` using message `Code executed on client` for Selector and Client function
Or some other way